.travis.yml

```diff
@@ -108,7 +108,26 @@ env:
       notification/mail.py
       notification/mqtt.py
       notification/sns.py
-      notification/typetalk.py"
+      notification/typetalk.py
+      storage/netapp_e/na_eseries_ethernet.py
+      storage/netapp_e/na_eseries_hostgroup.py
+      storage/netapp_e/na_eseries_amg_sync.py
+      storage/netapp_e/na_eseries_amg.py
+      storage/netapp_e/na_eseries_consistency_group.py
+      storage/netapp_e/na_eseries_snapshot_group.py
+      storage/netapp_e/na_eseries_snapshot_volume.py
+      storage/netapp_e/na_eseries_host.py
+      storage/netapp_e/na_eseries_facts.py
+      storage/netapp_e/netapp_storage_system.py
+      storage/netapp_e/na_eseries_volume.py
+      storage/netapp_e/na_eseries_snapshot_images.py
+      storage/netapp_e/na_eseries_lun_mapping.py
+      storage/netapp_e/na_eseries_flashcache.py
+      storage/netapp_e/na_eseries_controller_firmware.py
+      storage/netapp_e/na_eseries_storagepool.py
+      storage/netapp_e/na_eseries_auto_upgrade.py
+      storage/netapp_e/na_eseries_auth.py
+      storage/netapp_e/na_eseries_volume_copy.py"
```

Comment:
We're making an effort to get modules running on python3 as distributions are starting to ship without python2 by default.  We really shouldn't be adding more entries to this blacklist (it's not hard to get a module to **compile** under both python2 and python3.  Exception handling is the only thing that most modules need.)  Note that this test is only a compilation/syntax test.  It doesn't test that the module will run successfully.